packages/components/src/snackbar/stories/index.js

```diff
@@ -19,3 +19,14 @@ export const _default = () => {
 		</Snackbar>
 	);
 };
+
+export const withActions = () => {
+	const content = text( 'Content', 'Use Snackbars with an action link to an external page.' );
+	const actions = [ { url: text( 'URL', 'https://wordpress.org' ), label: text( 'label', 'Open WP.org' ) } ];
```

Comment:
We could experiment with the object knob as well: https://www.npmjs.com/package/@storybook/addon-knobs